h/static/scripts/directive/help-panel.js

```diff
@@ -11,15 +11,15 @@ module.exports = function () {
     bindToController: true,
     controllerAs: 'vm',
     // @ngInject
-    controller: function ($scope, $window, frameSync, serviceUrl) {
+    controller: function ($scope, $window, annotationUI, serviceUrl) {
       this.userAgent = $window.navigator.userAgent;
       this.version = '__VERSION__';  // replaced by versionify
       this.dateTime = new Date();
       this.serviceUrl = serviceUrl;
 
-      $scope.$watchCollection(
+      $scope.$watch(
```

Comment:
The identity of the array returned by `annotationUI.frames()` changes whenever the array is modified, so we can replace `$watchCollection` (which checks for changes between the old and the new value by comparing the contents of the list) with the slightly cheaper `$watch` (which only checks for changes in the value (ie. the identity in the case of an object/array).